packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -445,6 +446,90 @@ describe("Runtime", () => {
 						assert(submittedOps[1].metadata?.batchId === undefined, "Expected no batchId (1)");
 					}
 				});
+
+			for (const skipSafetyFlushDuringProcessStack of [true, undefined]) {
+				it("Inbound (non-runtime) op triggers flush due to refSeq changing", async () => {
```

Comment:
The scenario in this test doesn't seem possible with the current code. So, I would suggest adding a comment saying that is simulating a scenario where the delta manager doesn't call runtime for certain ops but emits an op event.